helm/hpcc/values.schema.json

```diff
@@ -1666,6 +1670,28 @@
         }
       }
     },
+    "topologySpreadConstraint": {
+      "type": "object",
+      "properties": {
```

Comment:
We should not try to clone the entire k8s schema. type: "object" is probably sufficient for any settings that are basically copied in to the helm unmodified.

Reply:
It may be helpful to add some properties into the schema that we expect people are likely to set, just because the schema MIGHT be used by a smart editor to give some hints. But we certainly should not enforce that ONLY those properties are set (for the reason Jake mentions)

Reply:
I can set it as an object but probably not bad if we can be notified something change which cause compatibility issues

Reply:
I think so long as we don't restrict additional settings, we are probably ok